openml-python-common/src/main/java/com/feedzai/openml/python/jep/instance/JepInstance.java

```diff
@@ -97,8 +101,18 @@ public void stop() {
      */
     @Override
     public void run() {
-
-        try (final Jep jep = new Jep(false)) {
+        final Set<String> sharedModules = ImmutableSet.<String>builder()
+                .add("tensorflow")
+                .add("numpy")
+                .addAll(new XMLParser().getSharedModules())
+                .build();
+        logger.debug("Python modules to be shared: {}", String.join(",", sharedModules.toString()));
+
+        final JepConfig jepConfig = new JepConfig()
+                .addSharedModules(sharedModules.toArray(new String[0]))
+                .setInteractive(false);
```

Comment:
what is this interactive?

Reply:
ok, got it: https://github.com/ninia/jep/blob/master/src/main/java/jep/JepConfig.java#L65

is this relevant for this to work or just a minor optimization? Asking because it seems it shouldn't be needed, and we didn't have it before.

Reply:
I'm not sure about the implication of changing this behaviour. I'm using the same behaviour used before this commit.

Reply:
oh, right, the interactive flag was passed directly to Jep constructor. you are right, forget my comment :)